logpush.go

```diff
@@ -10,14 +10,15 @@ import (
 
 // LogpushJob describes a Logpush job.
 type LogpushJob struct {
-	ID              int        `json:"id,omitempty"`
-	Enabled         bool       `json:"enabled"`
-	Name            string     `json:"name"`
-	LogpullOptions  string     `json:"logpull_options"`
-	DestinationConf string     `json:"destination_conf"`
-	LastComplete    *time.Time `json:"last_complete,omitempty"`
-	LastError       *time.Time `json:"last_error,omitempty"`
-	ErrorMessage    string     `json:"error_message,omitempty"`
+	ID                 int        `json:"id,omitempty"`
+	Enabled            bool       `json:"enabled"`
+	Name               string     `json:"name"`
+	LogpullOptions     string     `json:"logpull_options"`
+	DestinationConf    string     `json:"destination_conf"`
+	OwnershipChallenge string     `json:"ownership_challenge"`
```

Comment:
Do we want to omit this field when empty? I see a few of the operations don't have this field set (for example, getting a logpush job) where an empty string might be a touch confusing.

Reply:
Good point - I believe we do. Will update. Thanks for reviewing!